packages/api/lib/testUtils.js

```diff
@@ -7,7 +7,7 @@ const { createJwtToken } = require('./token');
 
 const isLocalApi = () => process.env.CUMULUS_ENV === 'local';
 
-const dataDir = 'app/data';
+const dataDir = `${process.cwd().split('cumulus')[0]}/cumulus/packages/api/app/data`;
```

Comment:
nice but won't work on my deployment.  my main cumulus repo is at `~/projects/cumulus/cumulus/packages/api/app/data` . 

I don't think we need to change this, because lerna's scope should handle this when running `npm run serve` from the top level.

Reply:
Looks like you're right.